WPMedia/fusion-news-theme-blocks
# TBRANDS-114 - Update to use API endpoint

## Description

Update Commerce Product content source to use API over mock data

## Jira Ticket

- [TBRANDS-114](https://arcpublishing.atlassian.net/browse/TBRANDS-114)

## Test Steps

1. Checkout this branch `git checkout TBRANDS-114-content-source-update`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/commerce-product-content-source-block,@wpmedia/product-gallery-block`
3. Using Fusion's debugger - http://localhost/pagebuilder/tools/debugger
4. Select a Site and Type
5. Then verify the content source "commerce-product" is show
6. Click on the content source "commerce-product"
7. Enter into the sku field - `indigo-5`
8. Click "Fetch"
9. Verify the response is as per required outcome (as per the old mock)
10. Testing the mock response
11. Enter into the sku field - `bounce-running-shoe`
12. Validate the data is as per the mock-data.js` file

Product Gallery Block Update

1. Create a new page
2. Add the product gallery block to the page
3. Set the global content to be
4. Content Source = commerce-product
5. SKU = `bounce-running-shoe` 
6. Validate the blue shoe pictures are loaded


## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
